documentation/source/eligible-collateral-representation.rst

```diff
@@ -209,7 +209,7 @@ or not.
 The combination of these terms allows a wide variety of eligible
 collateral types to be represented and structure can be used to identify
 individual collateral types or a group of collateral assets for
-inclusion in an eligible collateral schedule.
+inclusion in specifying eligible collateral schedule details.
```

Comment:
I assume you also want to remove 'schedule', i.e. just write 'in specifying eligible collateral details'?

Reply:
just to clarify did the graphic image upload get fixed too?

Reply:
Yes it did

Reply:
On taking second look this looks ok, it describes one of the use cases which is including details in a eligible collateral schedule, so this is ok

Reply:
Can we proceed with the contribution now to replace the document, thank you